### PR TITLE
Add warning message for slow exclusive vm request in verbose gc

### PR DESCRIPTION
**Pull request by LinHu2016**
- warning message shows the reason(JNICritical or Exclusive Access) 
	and time taken of slow exclusive request for performance diagnosis.

Signed-off-by: Lin Hu <linhu@ca.ibm.com>

### CODE REVIEW

File: runtime/gc_verbose_handler_standard_java/VerboseHandlerOutputStandardJava.cpp [hunk 1]
```diff
@@ -62,6 +63,8 @@ MM_VerboseHandlerOutputStandardJava::initialize(MM_EnvironmentBase *env, MM_Verb
 {
 	bool initSuccess = MM_VerboseHandlerOutputStandard::initialize(env, manager);
 	_mmHooks = J9_HOOK_INTERFACE(MM_GCExtensions::getExtensions(_extensions)->hookInterface);
+	J9JavaVM *javaVM = (J9JavaVM *)_extensions->getOmrVM()->_language_vm;
```
Comment (dmitripivkine) [inline runtime/gc_verbose_handler_standard_java/VerboseHandlerOutputStandardJava.cpp]: you can get OMR VM from env->getOmrVM(). Would it be cleaner?

File: runtime/gc_verbose_handler_standard_java/VerboseHandlerOutputStandardJava.cpp [hunk 2]
```diff
@@ -165,6 +170,24 @@ MM_VerboseHandlerOutputStandardJava::handleMarkEndInternal(MM_EnvironmentBase* e
 	}
 }
 
+void
+MM_VerboseHandlerOutputStandardJava::handleSlowExclusive(J9HookInterface** hook, UDATA eventNum, void* eventData)
+{
+	J9VMSlowExclusiveEvent* event = (J9VMSlowExclusiveEvent*)eventData;
```
Comment (dmitripivkine) [inline runtime/gc_verbose_handler_standard_java/VerboseHandlerOutputStandardJava.cpp]: Star should be after space: "J9VMSlowExclusiveEvent *event" - here and below

**Comment by dmitripivkine** [comment]
Would you please put a sample of output to PR description?

**Comment by dmitripivkine** [comment]
Jenkins sanity zLinux jdk8

**Comment by LinHu2016** [comment]
sample of output

```...
**<warning details="slow exclusive request due to Exclusive Access" threadname="JIT Compilation Thread-4" timems="95" />**
<exclusive-start id="195" timestamp="2018-04-09T15:35:25.987" intervalms="2833.579">
  <response-info timems="0.218" idlems="0.133" threads="12" lastid="0000000002C52600" lastname="Group1.Backend.CompositeBackend{Tier1}.7" />
</exclusive-start>
<af-start id="196" threadId="0000000002C5B260" totalBytesRequested="16" timestamp="2018-04-09T15:35:25.988" intervalms="2833.582" type="nursery" />
<cycle-start id="197" type="scavenge" contextid="0" timestamp="2018-04-09T15:35:25.988" intervalms="2833.583" />
<gc-start id="198" type="scavenge" contextid="197" timestamp="2018-04-09T15:35:25.988">
  <mem-info id="199" free="430206096" total="4294967296" percent="10">
    <mem type="nursery" free="0" total="2977955840" percent="0">
      <mem type="allocate" free="0" total="2402222080" percent="0" />
      <mem type="survivor" free="0" total="575733760" percent="0" />
    </mem>
    <mem type="tenure" free="430206096" total="1317011456" percent="32">
      <mem type="soa" free="364355728" total="1251161088" percent="29" />
      <mem type="loa" free="65850368" total="65850368" percent="100" />
    </mem>
    <remembered-set count="297003" />
  </mem-info>
</gc-start>
<allocation-stats totalBytes="2063035384" >
  <allocated-bytes non-tlh="72537752" tlh="1990497632" />
...```

**Comment by dmitripivkine** [comment]
Jenkins test sanity pLinux jdk8

**Comment by dmitripivkine** [comment]
Would you please fix type in the header of PR?

**Pull request merged**